diff -up minizip-ng-4.0.5/CMakeLists.txt.1~ minizip-ng-4.0.5/CMakeLists.txt
--- minizip-ng-4.0.5/CMakeLists.txt.1~	2024-03-11 17:14:36.450782918 +0100
+++ minizip-ng-4.0.5/CMakeLists.txt	2024-03-11 17:15:53.823668345 +0100
@@ -613,7 +613,7 @@ endif()
 
 # Include compatibility layer
 if(MZ_COMPAT)
-    set(SOVERSION "1")
+    #set(SOVERSION "1")
 
     set(FILE_H "ioapi.h")
     set(MZ_COMPAT_FILE "MZ_COMPAT_IOAPI")
