diff -up minizip-ng-3.0.1/CMakeLists.txt.omv~ minizip-ng-3.0.1/CMakeLists.txt
--- minizip-ng-3.0.1/CMakeLists.txt.omv~	2021-04-20 17:45:09.278705888 +0200
+++ minizip-ng-3.0.1/CMakeLists.txt	2021-04-20 17:49:32.236347364 +0200
@@ -198,18 +198,19 @@ endif()
 if(MZ_ZLIB)
     # Check if zlib is present
     if(NOT MZ_FORCE_FETCH_LIBS)
-        find_package(ZLIB QUIET)
+	    #        find_package(ZLIB QUIET)
         set(ZLIB_VERSION ${ZLIB_VERSION_STRING})
+	set(ZLIB_FOUND TRUE)
     endif()
 
     if(ZLIB_FOUND AND NOT MZ_FORCE_FETCH_LIBS)
         message(STATUS "Using ZLIB ${ZLIB_VERSION}")
 
         list(APPEND MINIZIP_INC ${ZLIB_INCLUDE_DIRS})
-        list(APPEND MINIZIP_LIB ${ZLIB_LIBRARIES})
+        list(APPEND MINIZIP_LIB -lz-ng)
         list(APPEND MINIZIP_LBD ${ZLIB_LIBRARY_DIRS})
 
-        set(PC_PRIVATE_LIBS " -lz")
+        set(PC_PRIVATE_LIBS " -lz-ng")
     elseif(MZ_FETCH_LIBS)
         clone_repo(zlib https://github.com/madler/zlib)
 
diff -up minizip-ng-3.0.1/mz_crypt.c.omv~ minizip-ng-3.0.1/mz_crypt.c
--- minizip-ng-3.0.1/mz_crypt.c.omv~	2021-04-20 17:44:35.471874695 +0200
+++ minizip-ng-3.0.1/mz_crypt.c	2021-04-20 17:44:49.071942164 +0200
@@ -14,9 +14,10 @@
 #include "mz_crypt.h"
 
 #if defined(HAVE_ZLIB)
-#  include "zlib.h"
-#  if defined(ZLIBNG_VERNUM) && !defined(ZLIB_COMPAT)
+#  if !defined(ZLIB_COMPAT)
 #    include "zlib-ng.h"
+#  else
+#    include "zlib.h"
 #  endif
 #elif defined(HAVE_LZMA)
 #  include "lzma.h"
diff -up minizip-ng-3.0.1/mz_strm_zlib.c.omv~ minizip-ng-3.0.1/mz_strm_zlib.c
--- minizip-ng-3.0.1/mz_strm_zlib.c.omv~	2021-04-20 17:42:29.617826077 +0200
+++ minizip-ng-3.0.1/mz_strm_zlib.c	2021-04-20 17:44:22.761810036 +0200
@@ -13,9 +13,10 @@
 #include "mz_strm.h"
 #include "mz_strm_zlib.h"
 
-#include "zlib.h"
-#if defined(ZLIBNG_VERNUM) && !defined(ZLIB_COMPAT)
+#if !defined(ZLIB_COMPAT)
 #  include "zlib-ng.h"
+#else
+#  include "zlib.h"
 #endif
 
 /***************************************************************************/
