diff -up minizip-ng-4.1.2/CMakeLists.txt.1~ minizip-ng-4.1.2/CMakeLists.txt
--- minizip-ng-4.1.2/CMakeLists.txt.1~	2026-04-25 07:50:35.000000000 +0200
+++ minizip-ng-4.1.2/CMakeLists.txt	2026-04-25 17:33:12.123171702 +0200
@@ -671,7 +671,7 @@ endif()
 
 # Include compatibility layer
 if(MZ_COMPAT)
-    set(SOVERSION "1")
+    #set(SOVERSION "1")
 
     list(APPEND MINIZIP_SRC
         compat/ioapi.c
