diff -up minizip-ng-4.0.8/CMakeLists.txt.1~ minizip-ng-4.0.8/CMakeLists.txt
--- minizip-ng-4.0.8/CMakeLists.txt.1~	2025-02-11 01:07:54.169073400 +0100
+++ minizip-ng-4.0.8/CMakeLists.txt	2025-02-11 01:08:24.477797223 +0100
@@ -615,7 +615,7 @@ endif()
 
 # Include compatibility layer
 if(MZ_COMPAT)
-    set(SOVERSION "1")
+    #set(SOVERSION "1")
 
     list(APPEND MINIZIP_SRC 
         compat/ioapi.c
